lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -309,3 +309,22 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
 function dosomething_kudos_disable_reactions($nid) {
   return !variable_get('dosomething_reportback_kudos_enabled', FALSE) || dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
 }
+
+function dosomething_kudos_get_kudos_data_for_fid($fid, $nid) {
```

Comment:
also the name of this function seems off, it does more than `get_kudos_data_for_fid` right?